boba_community/boba-node/docker-compose.yml

```diff
@@ -5,14 +5,12 @@ x-l1_rpc_dtl: &l1_rpc_dtl
 x-l1_rpc_geth: &l1_rpc_geth
   ETH1_HTTP: 'https://mainnet.infura.io/v3/YOUR_INFURA_KEY'
 
-version: "3"
+version: "3.9"
 
 services:
   dtl:
-    image: bobanetwork/data-transport-layer:latest
```

Comment:
The dtl needs to be built locally.

Reply:
why? it's silly. the DTL needs to be feed the state dump, that's no reason to re-build the entire image.

Reply:
Yes! Good idea.